File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        i , length = len(s) - 1, 0
        
        while s[i] == ' ':
            i -= 1
        while i >= 0 and s[i] != ' ':
            length += 1
            i -= 1
        return length
        

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int length = 0, i = s.length() - 1;
        
        while(s.charAt(i) == ' '){
            i--;
        }            
        while(i>=0 && s.charAt(i) != ' '){
            length++;
            i--;
        }
        return length;        
    }
}